packages/test-utils/src/reporter.ts

```diff
@@ -777,7 +777,7 @@ class ReplayReporter<TRecordingMetadata extends UnstructuredMetadata = Unstructu
           id: testRun.source.path + "#" + testRun.source.title,
           source: testRun.source,
           approximateDuration: testRun.approximateDuration,
-          recorded: true,
+          recorded: firstRecording != null,
```

Comment:
nit: I don't think we have to tolerate `null`s here as this comes from an element access
```suggestion
          recorded: firstRecording !== undefined,
```